cmd/eksctl-anywhere/cmd/validate.go

```diff
@@ -0,0 +1,166 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/config"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/providers/cloudstack/decoder"
+	"github.com/aws/eks-anywhere/pkg/validations"
+	"github.com/aws/eks-anywhere/pkg/workflows"
+)
+
+type validateOptions struct {
+	clusterOptions
+	skipIpCheck           bool
+	hardwareCSVPath       string
+	tinkerbellBootstrapIP string
+}
+
+var valOpt = &validateOptions{}
+
+var validateCmd = &cobra.Command{
+	Use:          "validate",
```

Comment:
I wonder if we should have a `eks-a exp validate create cluster` command
or at least `esk-a exp validate create`
I can def see us implementing another command for upgrades as well

Reply:
I definitely agree, will split this out